### PR TITLE
feat(metrics): Capture first-meaningful-paint for IssueList on Reload

### PR DESCRIPTION
**Pull request by leedongwei**
This PR is blocked by 

1. ~#18752: it is using `metric.mark` with the breaking API change~
2. ~getsentry/reload#162: it is using the new metric keys~

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/withOrganization.tsx [hunk 1]
```diff
@@ -29,4 +29,11 @@ const withOrganization = <P extends InjectedOrganizationProps>(
     }
   };
 
+export function isLightweightOrganization(
+  organization: Organization | LightWeightOrganization
+) {
```
Comment (markstory) [inline src/sentry/static/sentry/app/utils/withOrganization.tsx:34]: Should this function use `is` and be a [type predicate](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates)?

Comment (leedongwei) [reply src/sentry/static/sentry/app/utils/withOrganization.tsx]: Ooo this much better. Good catch.

**Review by markstory — APPROVED** [review]

**Comment by leedongwei** [comment]
Force-push to rebase on top of #18752

File: src/sentry/static/sentry/app/utils/withOrganization.tsx [hunk 2]
```diff
@@ -29,4 +29,11 @@ const withOrganization = <P extends InjectedOrganizationProps>(
     }
   };
 
+export function isLightweightOrganization(
+  organization: Organization | LightWeightOrganization
+): organization is LightWeightOrganization {
+  const castedOrg = organization as Organization;
+  return !(castedOrg.projects && castedOrg.teams);
+}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/withOrganization.tsx:37]: @leedongwei Is this reliable? Lightweight (currently) to the initial load of organization details, we defer loading projects/teams, but we re-attach it to the organization.

This is a temporary measure, but it's meant to maintain compatibility while we refactor components that depend on projects/teams on org.

Comment (leedongwei) [reply src/sentry/static/sentry/app/utils/withOrganization.tsx:37]: Are you concerned that the projects/teams might not be successfully fetched or attached to the organization?

Comment (billyvg) [reply src/sentry/static/sentry/app/utils/withOrganization.tsx:37]: @leedongwei oops nevermind, this is wrong - `projects/teams` do not get re-attached to the org.

**Pull request merged**